content/kubeflow/cleanup.md

```diff
@@ -0,0 +1,14 @@
+---
+title: "Cleanup"
+date: 2019-10-25T15:43:24-04:00
+weight: 90
+draft: false
+---
+
+### Uninstall Kubeflow
+
+Run these commands to uninstall Kubeflow from your EKS cluster
+```
+cd ${KF_DIR}
+kfctl delete -V -f ${CONFIG_FILE}
```

Comment:
I thought we had to remove Istio Ingress controller first and then delete all the resources?

Reply:
kfctl delete's everything now. Its neat!

Reply:
@arun-gupta I made the improvement in https://github.com/kubeflow/kubeflow/pull/4383. It's easier now.